experiment/autobumper/bumper/bumper.go

```diff
@@ -39,13 +40,33 @@ const (
 	testImageRepo   = prowRepo
 )
 
-func Call(cmd string, args ...string) error {
+func Call(stdout, stderr io.Writer, cmd string, args ...string) error {
 	c := exec.Command(cmd, args...)
-	c.Stdout = os.Stdout
-	c.Stderr = os.Stderr
+	c.Stdout = stdout
+	c.Stderr = stderr
 	return c.Run()
 }
 
+type SecretCensor interface {
+	Censor(content []byte) []byte
+}
+
+type HideSecretsWriter struct {
```

Comment:
nit: this does not need to be exported, and neither do its fields

Reply:
We do use this in the main pkg, also for the fields for init.

Reply:
Surprisingly, we can expose only the field name, but not the type.
But i feel odd that way. Would like to type exposed as well if it is OK.

Reply:
Ah, I thought we initialized in `Call()`. Nevermind.